.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["env", "react"]
+  "presets": ["env", "react", "es2017"]
```

Comment:
We can probably remove the es2017 preset since it should be covered by env.

Reply:
Yep. But we haven't solved yet the issue of using the spread operator outside of React components.